src/main/java/io/pravega/connectors/flink/serialization/PravegaSerialization.java

```diff
@@ -15,7 +15,12 @@
 /**
  * Helper methods to create DeserializationSchema and SerializationSchemas using the pravega JavaSerializer
  * for generic types.
+ *
+ * @deprecated PravegaSerialization is too non-specific about the serialization format that is used.
```

Comment:
In the javadocs, we should offer the alternative approach instead of explanation. 
```suggestion
 * @deprecated Please use the constructor with {@link JavaSerializer} instead.
```